tools/db_stress.cc

```diff
@@ -2645,8 +2645,9 @@ class StressTest {
 #else
       DBOptions db_options;
       std::vector<ColumnFamilyDescriptor> cf_descriptors;
-      Status s = LoadOptionsFromFile(FLAGS_options_file, Env::Default(),
-                                     &db_options, &cf_descriptors);
+      Status s =
+          LoadOptionsFromFile(FLAGS_options_file, Env::Default(), &db_options,
+                              &cf_descriptors, false, nullptr);
```

Comment:
since you are using default values, you can drop this change; or if you want to keep this change, please add comment for all values you are providing, like `&cf_descriptors, false /* ignore_unknown_options */, nullptr /* cache */)` to improve code readability.